### PR TITLE
OpenAI - Generate Text-to-Speech

### PR DESCRIPTION
**Pull request by srini047**
- Fixes: #1376 
- Made use of the Google Text to Speech API to generate the audio output of the OpenAI prompt response.

### CODE REVIEW

**Comment by Dr0p42** [comment]
No linked issues found. Please add the corresponding issues in the pull request description. <br/> [Use GitHub automation to close the issue when a PR is merged](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)

File: OpenAI/OpenAI_Generate_Text_to_Speech.ipynb [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import naas\n",
+    "\n",
+    "# OpenAI\n",
+    "try:\n",
+    "    import openai\n",
+    "except ModuleNotFoundError:\n",
+    "    !pip install openai\n",
```
Comment (Dr0p42) [inline OpenAI/OpenAI_Generate_Text_to_Speech.ipynb:94]: Hello @srini047, every `pip install` should use the `--user` parameter on naas.

File: OpenAI/OpenAI_Generate_Text_to_Speech.ipynb [hunk 2]
```diff
@@ -0,0 +1,262 @@
...
+    "    import openai\n",
+    "\n",
+    "# Google Text to Speech API\n",
+    "try:\n",
+    "    from gtts import gTTS\n",
+    "except ModuleNotFoundError:\n",
+    "    !pip install gTTS\n",
```
Comment (Dr0p42) [inline OpenAI/OpenAI_Generate_Text_to_Speech.ipynb:101]: Same here 👍

File: OpenAI/OpenAI_Generate_Text_to_Speech.ipynb [hunk 3]
```diff
@@ -0,0 +1,262 @@
...
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "response = openai.Completion.create(\n",
+    "  model=\"text-davinci-003\",\n",
+    "  prompt=prompt,\n",
+    "  temperature=0,\n",
```
Comment (Dr0p42) [inline OpenAI/OpenAI_Generate_Text_to_Speech.ipynb:172]: I think good basic parameters would be the same as the playground:
Temperature: 0.7 (We are getting good results with 0.5 as well)
Top_P:1.0
max_tokens: 256 (I really feel like 64 is too small)

**Review by Dr0p42 — COMMENTED** [review]
This seems promising 👍 looking forward to test it and integrate it in pipelines.

**Comment by FlorentLvr** [comment]
@srini047, that's awesome! The voice is still a bit robotic but I am sure they are going to improve it.

**Pull request merged**